test/Test/BaseDAO/Proposal.hs

```diff
@@ -231,6 +251,11 @@ flushNotAffectOngoingProposals :: (Monad m) => NettestImpl m -> m ()
 flushNotAffectOngoingProposals = uncapsNettest $ do
   ((owner1, _), _, dao, admin) <- originateBaseDaoWithConfig config
 
+  -- Note: Cannot set to few seconds, since in real network, each
+  -- calls takes some times to run. 20 seconds seem to be the ideal.
+  callFrom (AddressResolved admin) dao (Call @"Set_voting_period") 20
+
+  advanceTime (sec 3)
```

Comment:
It can wait for more than 3 seconds and in some cases it can wait for more than 20. AFAIU here the idea is to call `flush` before the voting period is over. Can we set voting period to a higher value, like 2 minutes for example?

Reply:
Make sense, I've updated this.